app/services/group_service.rb

```diff
@@ -17,8 +17,9 @@ def self.create(group:, actor:)
 
   def self.update(group:, params:, actor:)
     actor.ability.authorize! :update, group
-    group.update params
-    group
+    group.assign_attributes(params)
+    group.group_privacy = params[:group_privacy] if params[:group_privacy]
+    group.tap(&:save)
```

Comment:
yea.. that's cool. thanks for the reminder.

On Mon, 19 Oct 2015 at 12:06 James Kiesel <notifications@github.com> wrote:

> In app/services/group_service.rb
> <https://github.com/loomio/loomio/pull/2609#discussion_r42328111>:
>
> > @@ -17,8 +17,9 @@ def self.create(group:, actor:)
> >
> >    def self.update(group:, params:, actor:)
> >      actor.ability.authorize! :update, group
> > -    group.update params
> > -    group
> > +    group.assign_attributes(params)
> > +    group.group_privacy = params[:group_privacy] if params[:group_privacy]
> > +    group.tap(&:save)
>
> the tap-save is my new favorite thing.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/loomio/loomio/pull/2609/files#r42328111>.
>